src/hexo/themes/documentation/source/images/home-nellie.svg

```diff
@@ -1 +1 @@
:#333333;} .st55{fill:#F7E8CE;stroke:#2B2B2B;stroke-width:0.25;stroke-miterlimit:10;} .st56{stroke:#231F20;stroke-width:3;stroke-miterlimit:10;} .st57{fill:#4898D3;stroke:#2B2B2B;stroke-width:0.25;stroke-miterlimit:10;} .st58{clip-path:url(#SVGID_6_);} .st59{fill:#BA9066;stroke:#2B2B2B;stroke-width:0.25;stroke-miterlimit:10;} .st60{fill:#2B2B2B;stroke:#BA9066;stroke-width:0.25;stroke-miterlimit:10;} .st61{fill:#2D28A1;} .st62{fill:#EA664F;} .st63{fill:#EA5238;} .st64{fill:#D43E2B;} .st65{fill:#D33E2A;}</style><g id="Layer_1"><defs><path id="SVGID_3_" d="M193.9 173.2l-185.6-.6L9.8-29.7h184.1z"/></defs><clipPath id="SVGID_2_"><use xlink:href="#SVGID_3_" overflow="visible"/></clipPath><g class="st16"><path class="st2" d="M84 85.1c-37.3-3.6-70.7 26.5-74.7 67.3-.3 3.3-.4 6.5-.3 9.7 1 16.4 10.6 25.2 5.2 31.8L0 191.5c-6.2-2.7-27.3-17.4-32.1-18.1l.1-.2 1.9-3.3.8-1.3c3-5.1.4-12.4-5.8-16l-7.8-4.6c-.6-.4-1.4-.3-1.7.3l-7.7 13c-2 3.4-1.3 7.7 1.4 11.1l-.1.2c-4.3-.7-8.4.8-10.4 4.1l-7.7 13c-.3.5 0 1.3.6 1.6l7.8 4.6c6.2 3.7 13.8 2.5 16.8-2.7l1.4-2.4c4 6.3 11.5 14.1 20.5 21 2.4 1.8 4.3 3.1 7 4.8 8 5.3 16 9.4 23.1 12.1 12.6 5.2 28.3 8.2 45.2 8.2 16.3 0 31.4-2.8 43.7-7.7 5.1-1.9 10-4.3 14.5-7.4.9-.6 1.7-1.1 2.5-1.7.2-.2.4-.3.6-.5.5-.4.9-.7 1.4-1.1 15.3-12.2 26-31.1 28.2-53.1 4-40.6-23-76.6-60.2-80.3z"/><ellipse class="st3" cx="63.4" cy="124" rx="4.3" ry="3.7"/><ellipse class="st3" cx="110.5" cy="124" rx="4.3" ry="3.7"/><circle class="st4" cx="64.1" cy="123.6" r=".7"/><circle class="st4" cx="109.6" cy="123.6" r=".7"/><path class="st5" d="M87.6 143.4c-48.4 0-58.2 43.4-58.2 43.4-32.5 7.4-51.3 25-51.3 25 2.2 1.6 4.4 3.3 6.8 4.8 8 5.3 16 9.4 23.1 12.1 12.6 5.2 28.3 8.2 45.2 8.2 16.3 0 31.4-2.8 43.7-7.7 5.1-1.9 10-4.3 14.5-7.4.9-.6 1.7-1.1 2.5-1.7.2-.2.4-.3.6-.5.5-.4.9-.7 1.4-1.1 13.8-11 23.9-27.5 27.3-46.8-.3-1.6-8.3-28.3-55.6-28.3z"/><path class="st6" d="M82.1 136.8c2.9 2.2 6.9 2.2 9.9.1l.2-.1"/><path class="st5" d="M108 7.5L94.5 104l-.7 5.2c-2.9 2.3-7 1.9-9.8.3l-.5-.3L108 7.5z"/><path class="st17" d="M83.5 109.2s9-.7 11.1-5.7M86 100.8c2.6-.4 8.2-1.7 9.8-5.5M87.5 92.6c2.7-.5 7.8-1.9 9.3-5.4M89.9 83.9c2.8-.6 6.8-2.1 8.1-5.2M91.9 75.3c2.7-.7 6-2.2 7.2-4.9M93.6 67.1c2.6-.8 5.6-2.2 6.8-4.8M95.6 58.6c2.3-.8 4.8-2.2 5.8-4.5M97.7 50c2-.8 4-2.2 4.9-4.1M100.1 41.1c1.5-.8 3.2-2.5 3.8-4M105 28.9c-.5 1.2-1.8 2.6-2.9 3.3"/><path class="st2" d="M40.4 187.1s24.3-8.2 20.7-22.1c-.3-1.1-1.5-1.8-2.6-1.3l-23.8 8.6M129.7 184.2s-18.2-6.7-14.5-20.6c.3-1.1 1.5-1.7 2.6-1.3l21.2 8.9"/></g><path class="st21" d="M250.6 172.8h-44.8V91.5c0-3.4 2.8-6.2 6.2-6.2h32.3c3.4 0 6.2 2.8 6.2 6.2v81.3z"/><path class="st23" d="M-.3 172.8h262.9v14.7H-.3z"/><path class="st24" d="M136.1 170.5H59l-2.3 2.7h79.4z"/><path class="st21" d="M199.3 159.5H74.6c-.3 0-.5-.2-.5-.5V85.7c0-.3.2-.5.5-.5h124.6c.3 0 .5.2.5.5V159c0 .3-.2.5-.4.5z"/><path class="st25" d="M198.8 86.5h-8.2s-14.4 27.8-48 33.4c-37 6.2-64.7 38.4-64.7 38.4h120.8V86.5z"/><path class="st26" d="M142.4 167.9v-26.6h-7.9v26.6l-19.7 2.4v2.9H162v-2.9z"/></g></svg>
```

Comment:
@ststimac any reason why the Nellie in the home is different than the current version? It looks a lot bigger.